### PR TITLE
Replace outdated individal membership info with JavaScriptLandia #1196

### PR DESCRIPTION
**Pull request by PaulaPaul**
Draft for resolution of issue #1196  
(once more, with feeling)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,7 @@ Anyone who has been active in the foundation or one of its member projects, as d
 - Shelley Vohr ([@codebytere](https://github.com/codebytere))
 - Waleed Ashraf ([@waleedashraf](https://github.com/waleedashraf))
 - Yagiz Nizipli ([@anonrig](https://github.com/anonrig))
+- Paula Paul  ([@paulapaul](https://github.com/paulapaul))
```
Comment (joesepi) [inline README.md:186]: I suggest you move this to another PR and add yourself in alphabetical order. Thanks!

```suggestion

```

File: README.md [hunk 2]
```diff
@@ -182,7 +182,6 @@ Anyone who has been active in the foundation or one of its member projects, as d
 - Shelley Vohr ([@codebytere](https://github.com/codebytere))
 - Waleed Ashraf ([@waleedashraf](https://github.com/waleedashraf))
 - Yagiz Nizipli ([@anonrig](https://github.com/anonrig))
```
Comment (joesepi) [inline README.md:184]:
```suggestion
- Yagiz Nizipli ([@anonrig](https://github.com/anonrig))

```

**Review by PaulaPaul — COMMENTED** [review]
@joesepi let me know if I've 'unhosed' this -
